Tests/LibWeb/Layout/input/image-display-block-margin-auto.html

```diff
@@ -0,0 +1,13 @@
+<!doctype html><style>
+* {
```

Comment:
Nitpick: Could we remove this block? It doesn't change anything about the computed styling of the page and would reduce the size of this patch. Your layout test already assures the required values.